src/store/actions.ts

```diff
@@ -319,7 +319,7 @@ export const checkMRAS = ({ getters }, corpNum: string) => {
 export const fetchMRASProfile = async ({ commit, getters }): Promise<any> => {
   if (getters.getCorpSearch) {
     try {
-      let url = `mras-profile/${getters.getRequestJurisdictionCd}/${getters.getCorpSearch}`
+      let url = `mras-profile/${getters.getJurisdictionCd}/${getters.getCorpSearch}`
```

Comment:
`getRequestJurisdictionCd` was a duplicate getter, so I use the other getter here.